java/ql/lib/ext/com.amazonaws.auth.model.yml

```
@@ -0,0 +1,7 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["com.amazonaws.auth", "BasicAWSCredentials", False, "BasicAWSCredentials", "(String,String)", "", "Argument[0]", "credentials-key", "hq-generated"]
+      - ["com.amazonaws.auth", "BasicAWSCredentials", False, "BasicAWSCredentials", "(String,String)", "", "Argument[1]", "credentials-key", "hq-generated"]
```

Should these models (and all others that were originally in the `otherApiCallableCredentialParam` predicate) have a provenance of `"manual"` instead of `"hq-generated"`?

Unlike the other predicates, the `otherApiCallableCredentialParam` predicate did not have a comment saying `"Auto-generated using an auxiliary query run on the JDK source code."`, so I was under the impression that those models were manually created.

```suggestion
      - ["com.amazonaws.auth", "BasicAWSCredentials", False, "BasicAWSCredentials", "(String,String)", "", "Argument[0]", "credentials-key", "manual"]
      - ["com.amazonaws.auth", "BasicAWSCredentials", False, "BasicAWSCredentials", "(String,String)", "", "Argument[1]", "credentials-key", "manual"]
```

---

Good point. This is fixed now.